subworkflows/local/eggnog.nf

```diff
@@ -0,0 +1,39 @@
+//
+// Run eggnog-mapper on called ORFs, first optionally downloading the required databases
+//
+
+include { EGGNOG_DOWNLOAD } from '../../modules/local/eggnog/download'
+include { EGGNOG_MAPPER   } from '../../modules/local/eggnog/mapper'
+
+workflow EGGNOG {
+    take:
+        faa
```

Comment:
Doesn't have to be declared as `file faa` as one does in a module? (I haven't written a subworkflow for ages.)

Reply:
It should be only with `faa`